### PR TITLE
ASimNetworkSessionCiscoMeraki.yaml-36

### PR DESCRIPTION
**Pull request by Alekhya0824**
Required items, please complete
   
   Change(s):
   - Created a new file for Syslog and added Syslog parser in it. In meraki file removed Syslog parser and in ASimNetworkSession added Syslog Parser name  and same changes have done for vim as well.

   Reason for Change(s):
   - To Split the Syslog and meraki parser separately

   Version Updated:
   - yes

   Testing Completed:
   - yes

   Checked that the validations are passing and have addressed any issues that are present:
   - yes

### CODE REVIEW

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please check for failure validations. Thanks
image

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please respond on above. Thanks

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Parser:
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml:1]: why we are changing KQL tests?

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Network Session ASIM parser
-  Version: '0.7'
-  LastUpdated: Jan 17, 2024
+  Version: '0.7.2'
+  LastUpdated: June 5, 2024
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Network Session ASIM parser for Cisco Meraki
-  Version: '1.1.1'
-  LastUpdated: Mar 28 2024
+  Version: '1.2.1'
+  LastUpdated: June 5 2024
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml [hunk 4]
```diff
@@ -15,7 +15,7 @@ References:
 - Title: Cisco Meraki Documentation
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml:15]: correct url

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMerakiSyslog.yaml [hunk 5]
```diff
@@ -0,0 +1,386 @@
+Parser:
+  Title: Network Session ASIM parser for Cisco Meraki
+  Version: '1.2.1'
+  LastUpdated: June 5 2024
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMerakiSyslog.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMerakiSyslog.yaml [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: NetworkSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Network Session Schema
+  Link: https://aka.ms/ASimNetworkSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMerakiSyslog.yaml:14]: correct url

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Network Session ASIM filtering parser for Cisco Meraki
-  Version: '1.1.1'
-  LastUpdated: Mar 28 2024
+  Version: '1.2.1'
+  LastUpdated: June 5 2024
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml [hunk 8]
```diff
@@ -15,7 +15,7 @@ References:
 - Title: Cisco Meraki Documentation
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml:15]: asim url correct

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMerakiSyslog.yaml [hunk 9]
```diff
@@ -0,0 +1,454 @@
+Parser:
+  Title: Network Session ASIM filtering parser for Cisco Meraki
+  Version: '1.2.1'
+  LastUpdated: June 5 2024
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMerakiSyslog.yaml:4]: "LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMerakiSyslog.yaml [hunk 10]
```diff
@@ -0,0 +1,454 @@
...
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: NetworkSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Network Session Schema
+  Link: https://aka.ms/ASimNetworkSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
```
Comment (vakohl) [inline Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMerakiSyslog.yaml:14]: correct url

**Review by vakohl — CHANGES_REQUESTED** [review]

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml (hunk 1 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml:1]: I didn't got

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml (hunk 2 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml]: Done

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml (hunk 3 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml]: Done

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMerakiSyslog.yaml (hunk 5 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMerakiSyslog.yaml]: Done

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMerakiSyslog.yaml (hunk 6 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMerakiSyslog.yaml]: Done

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml (hunk 7 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml]: Done

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml (hunk 8 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml:15]: Done

File: Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMerakiSyslog.yaml (hunk 10 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMerakiSyslog.yaml]: Done

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
I can see a few open comments from the reviewer and PR is also having failed validations. Please act on them.
image
And also, one suggestion: once you're done with the open comment change, please click on the Resolve Conversation button.

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml (hunk 1 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml:1]: updating new table schema file

File: Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml (hunk 4 above)
Comment (Alekhya0824) [reply Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml:15]: done

**Comment by v-atulyadav** [comment]
Hi @Alekhya0824,
Please check above comment and respond accordingly. Thanks

**Review by vakohl — APPROVED** [review]

**Review by vakohl — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**